### PR TITLE
Block+Fighters

### PR DESCRIPTION
**Pull request by Tomash996**
first task Block

### CODE REVIEW

File: src/Fighter.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    public static String game(Fighter a, Fighter b) {
+
+        int counter = 0;
+        while (a.health > 0 && b.health>0) {
+
+            if (counter%2==0){
+                a.health = a.health - b.damagePerAttack;
+                if(a.health<=0){
+                    System.out.println(a.name + " - You're dead \n" + b.name + " is a winner");
```
Comment (vasylboliuk) [inline src/Fighter.java]: User String.format here

File: src/Fighter.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    return b.name;
+                }
+            } else {
+                b.health = b.health - a.health;
+                if(b.health<=0){
+                    System.out.println(b.name + " you're dead \n" + a.name + " is a winner");
```
Comment (vasylboliuk) [inline src/Fighter.java]: User String.format here

Comment (Tomash996) [reply src/Fighter.java]: done

File: src/Fighter.java (hunk 1 above)
Comment (Tomash996) [reply src/Fighter.java]: done

**Pull request merged**